.github/policies/resourceManagement.yml

```diff
@@ -182,7 +182,7 @@ configuration:
           label: teams-developer-support
       then:
       - addReply:
-          reply: "### Tell us about your experience! \n\nHi ${issueAuthor}! This issue is closed in our system. We would like your [feedback on your experience with our support team and Platform](https://aka.ms/DevSupportFeedback). \n\nBest regards, \nTeams Platform "
+          reply: "### Tell us about your experience! \n\nHi ${issueAuthor}! We've closed this issue in our system, but we'd love to hear your feedback. To let us know how we did, simply use one of the following emoji icons: :thumbsup: for resolved, :thumbsdown: for unresolved, or :heart: for a great experience.. Please use the [link for more detailed feeback](https://aka.ms/DevSupportFeedback). \n\nBest regards, \nTeams Platform "

```

Comment:
>simply use one of the following emoji icons [](http://example.com/codeflow?start=179&length=43)

Which message do you want them to use the icon on? Maybe consider "simply use one of the following emoji icons on this message"